tests/test_cli.py

```diff
@@ -222,6 +222,19 @@ def test_editable_package(tmpdir):
         assert 'six==1.10.0' in out.output
 
 
+def test_editable_package_vcs(tmpdir):
+    vcs_package = 'git+git://github.com/pytest-dev/pytest-django#egg=pytest-django'
```

Comment:
I would pin this to the current latest commit of `pytest-django` master branch.
The `pytest-django` package could stop using `setuptools_scm`.
Also, if they break the master branch for a while, I'd rather not have our tests fail on that.